scripts.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Scripts have been removed from the package.json file because we need the generator to handle changes to the package.json file. When the SDK is regenerated, the package.json file will be overwritten. We should not be adding anything to the package.json file.